component/loki/source/gelf/component.go

```diff
@@ -0,0 +1,147 @@
+package gelf
```

Comment:
Super silly nitpick, but should we rename these files to `gelf.go` and `gelf_test.go` so it's easier to navigate to like other components do?